### PR TITLE
Add esports match scraper

### PR DESCRIPTION
**Pull request by apinkert**
oh god oh fuck

image

Features:
 - Clears channel messages from previous day (only changes channel topic when there are no matches so should only get message pings on days with matches)
 - Runs every morning at 6:00am and gets matches within 24 hours (didn't wanna do day-of since game 1 of tokyo starts at 11pm and next at 2am)
 - Localized timestamps for match times
 - Match list is fragile as hell (probably will break if liquipedia breathes on their css classnames)

### CODE REVIEW

File: events/Scraper/scraper.js [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+  const messageManager = channel.messages;
+  const messages = await messageManager.channel.messages.fetch({ limit: amountToDelete });
+  channel.bulkDelete(messages,true);
+
+}
+
+//runs once a day at specified time 'SS MM HH' - 06:00 default
+client.once('ready', () => {
+  const channel = client.channels.cache.find(
+    channel => channel.name.toLowerCase() === "matches-for-today"
```
Comment (ryelo) [inline events/Scraper/scraper.js]: can you add this channel name to

https://github.com/ryelo/froggy/blob/main/config/guild.example.js

and then ref the `guild.js` + key in this file

File: yarn.lock
Comment (ryelo) [inline yarn.lock:1]: Delete this file

File: events/Scraper/scraper.js [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+    axios.get('https://liquipedia.net/valorant/Liquipedia:Matches').then(async matchResp => {
+      const matchArr = await MatchBuilder(matchResp.data, TourneyBuilder(tourneyResp.data));
+      if (matchArr.length > 0) {
+        channel.send({
+          embeds: [new EmbedBuilder()
+          .setColor('Green')
+          .setTitle('Matches for today')
+          ]
+        });
+        console.log('Matches scraped: ', matchArr);
```
Comment (ryelo) [inline events/Scraper/scraper.js]: extra log

**Review by ryelo — CHANGES_REQUESTED** [review]
haven't actually pulled the PR to go into more detail, but first glance few ezpz things

File: events/Scraper/scraper.js (hunk 1 above)
Comment (apinkert) [reply events/Scraper/scraper.js]: I've made the changes. Will the channel be automatically created in Froggin? Or does it require manual creation for `guild.js` to reference it?

Comment (ryelo) [reply events/Scraper/scraper.js]: You'll want to add a new entry to the [example file](https://github.com/ryelo/froggy/blob/main/config/guild.example.js)

import `guild.js` (not the example file)

and then just reference that key here.

Froggy will pick it up when I do the new deployment and add the new channel. It's just easier when we reference everything by the actual ID

**Review by ryelo — APPROVED** [review]

**Pull request merged**